[2024-12-06T21:59:55.340+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.is_weather_api_ready manual__2024-12-06T21:59:53.777779+00:00 [queued]>
[2024-12-06T21:59:55.345+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.is_weather_api_ready manual__2024-12-06T21:59:53.777779+00:00 [queued]>
[2024-12-06T21:59:55.345+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-12-06T21:59:55.353+0000] {taskinstance.py:2191} INFO - Executing <Task(HttpSensor): is_weather_api_ready> on 2024-12-06 21:59:53.777779+00:00
[2024-12-06T21:59:55.357+0000] {standard_task_runner.py:60} INFO - Started process 8489 to run task
[2024-12-06T21:59:55.359+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'is_weather_api_ready', 'manual__2024-12-06T21:59:53.777779+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/extract.py', '--cfg-path', '/tmp/tmpj5pnfdu9']
[2024-12-06T21:59:55.360+0000] {standard_task_runner.py:88} INFO - Job 62: Subtask is_weather_api_ready
[2024-12-06T21:59:55.371+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-12-06T21:59:55.414+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_dag.is_weather_api_ready manual__2024-12-06T21:59:53.777779+00:00 [running]> on host 8a5e48d7ee4d
[2024-12-06T21:59:55.507+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='thiernosidybah232@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='is_weather_api_ready' AIRFLOW_CTX_EXECUTION_DATE='2024-12-06T21:59:53.777779+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-06T21:59:53.777779+00:00'
[2024-12-06T21:59:55.508+0000] {http.py:130} INFO - Poking: /data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:59:55.516+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:59:55.776+0000] {base.py:295} INFO - Success criteria met. Exiting.
[2024-12-06T21:59:55.795+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=is_weather_api_ready, execution_date=20241206T215953, start_date=20241206T215955, end_date=20241206T215955
[2024-12-06T21:59:55.853+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-12-06T21:59:55.881+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
